crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -176,6 +183,17 @@ impl<'a, S: Scalar> Column<'a, S> {
                     alloc.alloc_slice_copy(scalars.as_slice()),
                 ))
             }
+            OwnedColumn::VarBinary(col) => {
+                let scalars = col
+                    .iter()
+                    .map(|b| S::from_le_bytes_mod_order(b))
+                    .collect::<Vec<_>>();
+                let bytes = col.iter().map(|s| s as &'a [u8]).collect::<Vec<_>>();
+                Column::VarBinary((
+                    alloc.alloc_slice_clone(&bytes),
+                    alloc.alloc_slice_copy(scalars.as_slice()),
+                ))
+            }
             OwnedColumn::TimestampTZ(tu, tz, col) => Column::TimestampTZ(*tu, *tz, col.as_slice()),
```

Comment:
This is insecure because it is really easy to find collisions.
[0] is the same as [0,0,0,0,0] is the same as the le bytes of the field modulus as a "`u256`"  is the same as the le bytes of 2x the field modulus as a "`u256`".

Instead, we need to hash.